### PR TITLE
Update android.md

### PR DESCRIPTION
**Pull request by c-tshibas**
Adding context on crash reporting for RUM

### What does this PR do? What is the motivation?

Adding context on crash reporting for RUM for the Android SDK and its behaviours

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by buraizu** [comment]
Thanks, created DOCS-8648 to review

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 1]
```diff
@@ -259,6 +259,15 @@ To test your implementation:
 
 3. After the crash happens, restart your application and wait for the Android SDK to upload the crash report in [**Error Tracking**][1].
 
+## Behaviours
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:262]:
```suggestion
## Behaviors
```

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 2]
```diff
@@ -259,6 +259,15 @@ To test your implementation:
...
+
+When looking at the behaviours of the RUM Crash Reporting for Android, there are some points to consider:
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:264]:
```suggestion
When looking at RUM Crash Reporting behaviors for Android, consider the following:
```

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 3]
```diff
@@ -259,6 +259,15 @@ To test your implementation:
...
+
+- The crashes can only be detected after the Datadog SDK is initialised, which is why it is recommended to initialize the SDK as soon as possible in the application's `onCreate` method
+- Crashes in RUM need to be tied to a RUM View. This means that if a crash occurs before a view is visible, or after the app is sent to the background and the end-user switches to another app, the crash will be muted and won't report to Datadog. This behavior can be mitigated using the `trackBackgroundEvents` [method](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/setup/android/?tab=kotlin#track-background-events) in the RumConfiguration builder.
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:267]:
```suggestion
- Crashes can only be detected after the Datadog SDK is initialized. To catch as many crashes as possible, initialize the SDK as soon as you can in the application's `onCreate` method.
- Crashes in RUM need to be tied to a RUM View. This means that if a crash occurs before a view is visible, or after the app is sent to the background and the end-user switches to another app, the crash is muted and does not report to Datadog. Mitigate this behavior using the `trackBackgroundEvents` [method][10] in the RumConfiguration builder.
```

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 4]
```diff
@@ -259,6 +259,15 @@ To test your implementation:
...
+- Also crashes tied to sampled sessions will only be kept, meaning if a `sessionSampleRate` of less than 100% is applied, some crashes will be missed.
```
Comment (hestonhoffman) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:268]:
```suggestion
- Only crashes tied to sampled sessions are kept. To capture all crashes, apply a `sessionSampleRate` of 100%.
```

**Review by hestonhoffman — COMMENTED** [review]
@c-tshibas I left some edits for you. Mostly just US spelling and got rid of some future tense / passive voice

**Pull request merged**